temporalio/worker/_workflow_instance.py

```diff
@@ -1791,6 +1800,42 @@ def _stack_trace(self) -> str:
             )
         return "\n\n".join(stacks)
 
+    def _enhanced_stack_trace(self) -> temporalio.common._EnhancedStackTrace:
```

Comment:
To clarify, even just showing us what this may look like here in GH comments can help us understand how good/bad the Python stack traces may be. I recall they are pretty bad for many users.

Reply:
[stack_trace.txt](https://github.com/user-attachments/files/15774955/stack_trace.txt)
This is the test output for a `__stack_trace` call, for reference!

Reply:
Thanks! Yeah that does a good job of showing the current poor state of Python stack traces. We'll confer as a team to see what we can do here (and if we want enhanced to wait on that).